examples/aspnetcore-with-serilog/Controllers/WeatherForecastController.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+
+namespace AspnetCoreExample.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class WeatherForecastController : ControllerBase
```

Comment:
Think this should be made the default route for the app, so this controller is hit on startup.

Reply:
++

Reply:
On second thoughts, I want to keep this as close to the vanilla project as shipped by VS 2019, in order to reduce the comprehension overhead. Will mark as resolved.